packages/proposals

```diff
@@ -0,0 +1,53 @@
+1 PKG_PRIORITY
+
+None of the current scripts use the PKG_PRIORITY variable.
+All but three packages set PKG_PRIORITY="optional". busybox, dbus and systemd
+set PKG_PRIORITY="required".
+The effect of PKG_PRIORITY is unknown.
+
+Proposal:
+Deprecate the PKG_PRIORITY variable
+
+
+2 PKG_SITE, PKG_LONGDESC, PKG_SHORTDESC
+
+None of the current scripts use the PKG_SITE variable.
+create_addon is the only script that uses the PKG_LONGDESC and PKG_SHORTDESC
+variables.
+In other words, the PKG_LONGDESC and PKG_SHORTDESC are only used for
+addons. For other packages, the PKG_SITE, PKG_LONGDESC and PKG_SHORTDESC are
+merely informative and redundant.
+
+Proposal:
+Deprecate PKG_LONGDESC and/or PKG_SHORTDESC for normal packages
+
```

Comment:
A valid PKG_SITE is most definitely useful - as pointed out it can be separate from the download location, and saves a lot of searching which may not always turn up the canonical project.

We probably don't need a long AND a short description for non-addon packages, but all packages should have at least one vaguely appropriate description (PKG_LONGDESC?) as it provides a quick overview of what the package is about which is very useful for beginners that may not be familiar with the many hundreds of packages we use.